sql/moz-fx-data-marketing-prod/google_search_console_derived/url_impression_exports_empty_check_v1/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: URL Impression Exports Empty Check
+description: |-
+  Raises an error if any of the Google Search Console URL impression exports are missing data for the given date.
+owners:
+- srose@mozilla.com
+scheduling:
+  dag_name: bqetl_google_search_console
+  destination_table: null
+  query_file_path: sql/moz-fx-data-marketing-prod/google_search_console_derived/url_impression_exports_empty_check_v1/query.sql
+  # Google Search Console exports typically happen two days after the data date in UTC.
+  parameters:
```

Comment:
For consistency should we do it like this?
https://github.com/mozilla/bigquery-etl/pull/4892/files#diff-ee64d2d3510eed266e548988757bc4c5b0b39871cfe551fa9118d9014ad2b4dbR23

Reply:
See [other thread](https://github.com/mozilla/bigquery-etl/pull/4892#discussion_r1481999013).